core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250228_add_billing_project_id.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.30.xsd">
+    <changeSet id="add_billing_project_id" author="kmarete" logicalFilePath="dummy">
+        <addColumn tableName="BILLING_PROJECT">
+            <column name="ID" type="BINARY(16)">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+
+        <!-- Populate ID column with UUID derived from NAME (Unique PK) -->
+        <update tableName="BILLING_PROJECT">
+            <column name="ID" valueComputed="UNHEX(MD5(NAME))"/>
+        </update>
+    </changeSet>
```

Comment:
1. Do you need separate `<addColumn>` and `<update>` operations? Can you just add the `valueComputed="UNHEX(MD5(NAME))"` attribute to the column inside the `<addColumn>` operation and do away with the `<update>` entirely?
2. Will this allow us (TDR) to insert rows and specify an exact UUID? We will need that functionality to copy over TDR spend profiles. I'm wondering if you need to use `defaultValueComputed` instead of `valueComputed`

Reply:
Fixed 1: https://github.com/broadinstitute/rawls/pull/3205/commits/f2554e9b7323ef1d111bba3cef44fa77dfade619 and 2: https://github.com/broadinstitute/rawls/pull/3205/commits/40daaf845287660732d21f9e2cf33adaaca1bcd0

Reply:
No need because we add a random UUID when creating billing projects: https://github.com/broadinstitute/rawls/blob/a3f2d04f283d9f061ccbcc5720d704db3d2caeef/core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProjectOrchestrator.scala#L163